File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.py
def solution(l, r):
    answer = []
    for i in range(1,64,1):
        if l <= int(bin(i)[2:])*5 <= r:
            answer.append(int(bin(i)[2:])*5)
    if not answer:
        answer.append(-1)
    
    return answer

File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.java
import java.util.*;

class Solution {
    public int[] solution(int l, int r) {
        List<Integer> answer = new ArrayList<Integer>();
        
        for(int i=1;i<64;i++){
            int num = Integer.valueOf(Integer.toBinaryString(i)) * 5;
            if(num >= l && num <= r)
                answer.add(num);
        }
        if(answer.size()==0) answer.add(-1);
        return answer.stream().mapToInt(i->i).toArray();
    }
}